docs/src/main/asciidoc/continuous-testing.adoc

```diff
@@ -150,6 +150,13 @@ mode run `mvn quarkus:test`.
 
 NOTE: The Dev UI is not available when running in continuous testing mode, as this is provided by dev mode.
 
+== Multi Module Projects
+
+Note that continuous testing supports multi-module projects, so tests in modules other than the application can still
+be run when files are changed. The modules that are run can be controlled using config as listed below.
```

Comment:
> The modules that are run can be controlled using config as listed below.

Isn't there just that one toggle? Reads as if I could exclude/include specific modules...?

Reply:
Ah, forget about that! This is refererring to the config properties that are rendered automatically. 🤦‍♂️